backend/pipeline/validator.py

```diff
@@ -1,12 +1,15 @@
 import xml.etree.ElementTree as ET
+from utils.minio import client
 
 from .rules.parquet_file_output_rule import ParquetFileOutputRule
 # A XML Schema validation check should be implemented in the future to ensure that the XML data is valid
-def check_pipeline_validity(name):
+def check_pipeline_validity(name, user_id=None):
```

Comment:
do we need the user_id here?

Reply:
We need this to browse minio, because the file won't be saved in local file system